tuna/worker_interface.py

```diff
@@ -398,7 +387,7 @@ def check_env(self) -> bool:
       raise ValueError(
           f'session rocm_v {self.dbt.session.rocm_v} does not match env rocm_v {env_rocm_v}'
       )
-    env_miopen_v: str = self.get_miopen_v()
+    env_miopen_v: str = chk_miopen_env_v(self)
```

Comment:
miopen_v should not be present in worker_interface.

Reply:
I will rename the API name. which would be referred from fin_class. pls confirm

Reply:
You can remove this line, if there is some feature elsewhere in this class that refers to env_miopen_v, then that will need to be addressed as well.